src/Arcus.Security.Providers.KeyPerFile/KeyPerFileSecretProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Arcus.Security.Core;
+using GuardNet;
+using Microsoft.Extensions.Configuration.KeyPerFile;
+using System;
+using System.Threading.Tasks;
+
+namespace Arcus.Security.Providers.KeyPerFile
+{
+    public class KeyPerFileSecretProvider : ISecretProvider
```

Comment:
```suggestion
    /// <summary>
    /// Represents an <see cref="ISecretProvider" /> that provides access to the Docker secrets mounted into the Docker container as files.
    /// </summary>
    public class KeyPerFileSecretProvider : ISecretProvider
```

Provide some summary for the secret provider.

Reply:
Done.   I did it via VS.NET instead of committing the suggestion, since I've renamed some files and directories.